src/Card-Description.js

```diff
@@ -1,17 +1,65 @@
 import React, { Component } from 'react';
 import cardDescriptionStyle from './Card-Description.scss';
+import Rescue from './Rescue.js';
 
 export default class CardDescription extends Component {
+  constructor() {
+    super();
+    this.state = {
+      displayDescription: true,
+      currentRescue: []
+    }
+  }
+
+  toggleRescue = (event) => {
+    this.filterRescue();
+    this.setState({displayDescription: !this.state.displayDescription});
+  }
+
+  filterRescue = () => {
+    let matchedRescue = this.props.rescues.find((currentRescue) => {
+      return this.props.description.rescue.toLowerCase() === currentRescue.name.toLowerCase()
+    })
+    console.log('matchedRescue', matchedRescue);
+    this.state.currentRescue.push(matchedRescue);
+    // this.displayRescue(matchedRescue);
+  }
+
+  // displayRescue = (matchedRescue) => {
+
+  //   return (
+  //     <Rescue className='rescue' matchedRescue={matchedRescue} />
+  //   }
+  // }
+
```

Comment:
Avoid committing commented out code and console logs, it makes the PRs more difficult to read and review.